要测试的文件（myprogram.py）：

```python
def get_formatted_name(first, last):
    """Generate a neatly formatted full name."""
    full_name = first + " " + last
    return full_name.title()
```

测试的文件（test.py）：

```python
import unittest
from myprogram import get_formatted_name

class NamesTestCase(unittest.TestCase):

    def test_first_last_name(self):
        formatted_name = get_formatted_name("janis", "joplin")
        self.assertEqual(formatted_name, 'Janis Joplin')


unittest.main()
```

确保这两个文件在同一个目录下，可以使用如下命令执行测试文件：

```console
$ python test.py
```

> 在执行测试文件时，不需要将 myprogram.py 加到命令中，否则会如下报错：
> ```console
$ python test.py myprogram.py
Traceback (most recent call last):
  File "test.py", line 100, in <module>
    unittest.main()
  File "C:\Python27\lib\unittest\main.py", line 94, in __init__
    self.parseArgs(argv)
  File "C:\Python27\lib\unittest\main.py", line 149, in parseArgs
    self.createTests()
  File "C:\Python27\lib\unittest\main.py", line 158, in createTests
    self.module)
  File "C:\Python27\lib\unittest\loader.py", line 130, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Python27\lib\unittest\loader.py", line 100, in loadTestsFromName
    parent, obj = obj, getattr(obj, part)
AttributeError: 'module' object has no attribute 'myprogram'
> ```